src/models/changeLog/selectors.ts

```diff
@@ -4,6 +4,6 @@ export const selectSize = (state: RootStateType): number | undefined => {
   return state.changesLog.size;
 };
 
-export const selectTotalPages = (state: RootStateType): number | undefined => {
+export const selectTotalPages = (state: RootStateType): number | unknown => {
```

Comment:
why `unknown`?

Reply:
Sorry my fault 
Im change it on undefined